src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,12 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100vw;
+  justify-content: space-between;
+  max-inline-size: calc((#{$card-width} * 4) + (#{$gap} * 3));
+}
```

Comment:
![2023-05-17_15h44_20](https://github.com/mate-academy/layout_catalog/assets/61904995/255dc776-87f4-4d0f-a90d-5c1e61460925)
It should be always 4 crads
![2023-05-17_15h18_36](https://github.com/mate-academy/layout_catalog/assets/61904995/9d83616c-c8dd-4255-bf94-51e85ad34498)